packages/pyright-internal/src/tests/samples/typeForm4.py

```diff
@@ -2,29 +2,12 @@
 
 # pyright: reportMissingModuleSource=false
 
-from typing import (
-    Annotated,
-    Any,
-    Callable,
-    Concatenate,
-    Final,
-    Generic,
-    Literal,
-    LiteralString,
-    Never,
-    NewType,
-    NoReturn,
-    NotRequired,
-    Optional,
-    Required,
-    Type,
-    TypeAlias,
-    TypeGuard,
-    TypeVar,
-    Union,
-    Unpack,
-)
 import typing
+from typing import (Annotated, Any, Callable, Concatenate, Final, Generic,
```

Comment:
This is strange. I'm not sure why this happened. Maybe this is an older version of typeForm4.py that we're pushing back? Maybe this is related to the `parent` commit in our `.gitrepo` being out of date?

I think we should undo this.

Reply:
sure